Auto/Translation/LamFOL2SMT.lean

```diff
@@ -429,21 +453,23 @@ def termAuxDecls : Array IR.SMT.Command :=
   `valid_fact_{i}` corresponds to the `i`-th entry in `facts`
 -/
 def lamFOL2SMT
-  (lamVarTy lamEVarTy : Array LamSort)
+  (lamVarTy : Array (Expr × LamSort))
```

Comment:
Inappropriate naming across entire file! Should be named ``varVal`` because it contains the valuation of the term.

Reply:
Valuation for term atoms should be eta-reduced. This is likely an issue irrelevent to this pull request.